### ISSUE TITLE
'Grace period' for blade fury reapplying

### ISSUE THREAD

**Issue by Soybeanpaste** [Hekili/hekili#2216]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/enhancement) for this request and was not able to find one.
- [X] I edited the title of this feature request (above) so that it describes the issue I am reporting.

### WoW Version

Retail (Dragonflight)

### Feature Request

For Rogue AoE, it's very important to have a 100% uptime for Blade Furry. 

Sometimes with the 'windfury' effect we have from Sinister Strike, Ambush or Pistol shot it suggests us to use these abilities while the blade furry would still be up during when you press the ability, but when it's almost about to run out, I've noticed that it doesn't blade fury all these hits (due to small delays/lag/latency), is there a way to make blade fury have a 'grace period' of 1 GCD extra so this doesn't happen? 

### Additional Information

So basically i'm asking that we have an option to have blade fury be listed in the prioity 1-2 GCD before it expires so the above doesn't happen :)

**Comment by Soybeanpaste** [comment]
I don't know why it has the label enhancement and how to change it to rogue sorry

**Comment by Hekili** [comment]
You can copy and edit the priority to do that, if you'd like.  The sim profile for Outlaw pretty explicitly waits for Blade Flurry to fall off before reapplying it.  The first entry of the `cds` action list is for Blade Flurry, and checks that `! buff.blade_flurry.up`.  You could change that to `buff.blade_flurry.remains < gcd.max` to get what you're looking for.
